server/templates/login.html

```diff
@@ -16,63 +16,89 @@
+		}
+	</style>
+
 </head>
-<body>
-<form method="post" action="/login">
-  <label for="program">Program:</label><br>
-  <select name="program" id="program">
-    <option value="">Select...</option>
-    <option value="FTC">FIRST Tech Challenge</option>
-    <option value="FRC">FIRST Robotics Competition</option>
-  </select>
-  <br></br>
-
-  <label for="teamNumber">Team Number:</label><br>
-  <input name="team_number" id="teamNumber" type="text" style="width: 20ch" value="{{team_number}}">
-  <br></br>
-
-  <label for="teamCode">Team Code:</label><br>
-  <input name="team_code" id="teamCode" type="password" style="width: 20ch">
-  <br></br>
-
-  <input id="submitButton" type="submit" value="Submit">
-</form>
-<br>
-<div class="errorMessage">{{error_message}}</div>
-
-<script type="text/javascript">
-  updateSubmitButton = function() {
-    const programSelect = document.getElementById('program');
-    const teamNumberInput = document.getElementById('teamNumber');
-    const teamCodeInput = document.getElementById('teamCode');
-    const submitButton = document.getElementById('submitButton');
-    submitButton.disabled = (
-        programSelect.selectedIndex < 1 ||
-        !teamNumberInput.value ||
-        !teamCodeInput.value);
-  };
-
-  window.addEventListener('load', function() {
-    const programSelect = document.getElementById('program');
-    const teamNumberInput = document.getElementById('teamNumber');
-    const teamCodeInput = document.getElementById('teamCode');
-
-    for (let i = 0; i < programSelect.options.length; i++) {
-      if (programSelect.options[i].value == '{{program}}') {
-        programSelect.selectedIndex = i;
-        break;
-      }
-    }
-
-    updateSubmitButton();
-    programSelect.onchange = updateSubmitButton;
-    teamNumberInput.oninput = updateSubmitButton;
-    teamCodeInput.oninput = updateSubmitButton;
-  });
-</script>
+<body class="text-center">
+
+	<main class="form-signin text-center">
+		<form method="post" action="/login">
+			<img class="mb-4" src="https://www.firstinspires.org/sites/all/themes/first/assets/images/2020/first-horz-rgb.png" alt="" style="width: 250px; height: auto;">
+			
+			<h1 class="h3 mb-3 fw-bold">First Machine<br>Learning Toolchain</h1>
+			<h2 class="h2 mb-3 fst-italic">Version 0.0001</h2>
+
+			<div class="form-floating">              
+				<select name="program" id="program" class="form-control">
+					<option value="">Select...</option>
+					<option value="FTC">FIRST Tech Challenge</option>
+					<option value="FRC">FIRST Robotics Competition</option>
+				</select>
+				<label for="program">Program:</label>
+			</div>
+
+			<div class="form-floating">
+				<input type="text" class="form-control" name="team_number" id="teamNumber" placeholder="" value="{{team_number}}">
+				<label for="teamNumber">Team Number</label>
+			</div>
+			<div class="form-floating">
+				<input type="password" name="team_code" class="form-control" id="teamNumber" placeholder="">
```

Comment:
This should be ...id="teamCode"...

Reply:
Yeah... just hanging my head in shame here. Fixed though.